pkg/cmd/create/work/cmd.go

```diff
@@ -22,8 +22,8 @@ func NewCmd(clusteradmFlags *genericclioptionsclusteradm.ClusteradmFlags, stream
 
 	cmd := &cobra.Command{
 		Use:          "work",
-		Short:        "create a work",
-		Long:         "create manifestwork on a specified managedcluster",
+		Short:        "create a work using resource-to-apply yaml file",
+		Long:         "create a work using resource-to-apply yaml file. The resource-to-apply yaml file is exactly the yaml file you want to apply on the managed cluster. Take care and do not use a ManifestWork yaml file here.",
```

Comment:
using a file containing common kubernetes resource manifests, or a director containing a set of manifest files. 

We do not need to say don't use ManifestWork file, it is a bit confusing.